Small typo edits

hharveygit opened the pull request:
- Added a minimum requirement for 2 channels in ccep_CARLA (consistent with performance in manuscript repo)
- Fixed a small display typo in ccep_CARVariance (displays "s" instead of "ms")
- Comment changes
- Added to .gitignore to ignore local deprecated directories

hharveygit committed:
Small fixes

ccep_CARLA is guaranteed now to start with min cluster size = 2.
Minor comment changes

hharveygit committed:
Merge remote-tracking branch 'upstream/master'

Pulling from upstream to get wang colormap code

hharveygit committed:
Minor display typo edit: ms to s

Responsive window was being displayed to screen in "ms" instead of "s", which is correct. This is fixed now.

hharveygit committed:
Updated ieeg_mni305ThroughFsSphere.m

ieeg_mni305ThroughFsSphere.m
- added surfaceChoice, distLim optional arguments (to choose which surface pial/white and maximum distance
- now returns vertIdxFsavg, minDists, surfUsed additional output arguments, for diagnostic purposes and to easily get matching inflated/sphere coordinates
- code remains backwards compatible

dorahermes commented on functions/ieeg_mni305ThroughFsSphere.m:
```diff
@@ -1,18 +1,26 @@
 % it should ony be used for ECoG.
 %
+% mni_coords = ieeg_mni305ThroughFsSphere(elecmatrix, hemi, FSdir, FSsubjectsdir); % minimum inputs/outputs
+% [mni_coords, vertIdxFsavg, minDists, surfUsed] = ieeg_mni305ThroughFsSphere(elecmatrix, hemi, FSdir, FSsubjectsdir, surfaceChoice, distLim); % maximum inputs/outputs
```
use to varargin or something instead?

hharveygit committed:
Updated ieeg_mni305ThroughFsSphere.m

ieeg_mni305ThroughFsSphere.m
- minor fix with nargins so surfaceChoice, distLim inputs can be not given
